src/components/buttons/back-button/tests/back-button.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import * as reactRedux from 'react-redux';
+import Adapter from 'enzyme-adapter-react-16';
+import BackButton from '../index';
+import { config } from '../../../../configs';
+import useSuccessSnackbar from '../../../../utils/use-success-snackbar';
+import { variables as testProps } from './variables';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const { GO_BACK_TITLE } = config.buttonTitles;
+
+const { openSuccessSnackbar } = useSuccessSnackbar();
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useDispatch: () => jest.fn()
+}));
+
+describe('Back button tests', () => {
+  const useDispatchMock = jest.spyOn(reactRedux, 'useDispatch');
+  let component;
+
+  jest.spyOn(reactRedux, 'useDispatch');
+  const mockHolder = jest.fn();
+
+  beforeEach(() => {
+    component = mount(<BackButton {...testProps} />);
+  });
+  afterEach(() => {
+    component.unmount();
+  });
+  it('Should handle click the button', () => {
+    useDispatchMock.mockReturnValue(mockHolder);
+    const wrapper = mount(<BackButton />);
+    wrapper.find('button').simulate('click');
+    expect(mockHolder.mock.calls.length).toBeDefined();
```

Comment:
wrapper.find('button').simulate('click') does not change mockHolder.mock.calls.length